src/js/components/Popup.js

```diff
@@ -13,30 +15,44 @@ const Popup = forwardRef((props, ref) => {
     }
   }
 
-  const serializedProps = serializeProps([
-    "loading",
-    "service",
-    "subdomain",
-    "projects",
-    "activities",
-    "schedules",
-    "timedActivity",
-    "serviceLastProjectId",
-    "userLastProjectId",
-    "serviceLastTaskId",
-    "userLastTaskId",
-    "fromDate",
-    "toDate",
-    "errorType",
-    "errorMessage",
-  ])(props)
+  const sendData = useCallback(() => {
```

Comment:
Needs a `useCallback` because this function is a dependency in a `useEffect`, in L43 in this file. Without the `useCallback` here, the useEffect is fired each time the component renders.